packages/blocks/src/__internal__/rich-text/keyboard.ts

```diff
@@ -44,6 +45,7 @@ type KeyboardBinding = {
    * See https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values
    */
   key: number | string | string[];
+  collapsed?: boolean;
```

Comment:
Why using `satisfies` could match the type here without adding this field?

Reply:
Yes, It's unexpected.